Resources/views/CRUD/base_list.html.twig

```diff
@@ -17,6 +17,21 @@ file that was distributed with this source code.
 
 {% block tab_menu %}{{ knp_menu_render(admin.sidemenu(action), {'currentClass' : 'active', 'template': sonata_admin.adminPool.getTemplate('tab_menu_template')}, 'twig') }}{% endblock %}
 
+{% block title %}
+    {#
+        The list template can be used in nested mode,
+        so we define the title corresponding to the parent's admin.
+    #}
+    
+    {% if admin.isChild and admin.parent.subject %}
+        {{ "title_edit"|trans({'%name%': admin.parent.toString(admin.parent.subject)|truncate(15) }, 'SonataAdminBundle') }}
```

Comment:
The truncate filter is going to disappear, so this should be replaced anyway.

> You can't guess, what the browser tab is all about, if we don't truncate the entity title at all.

If this is b/c the title is in the middle of the translation, maybe put it at the end?